static/css/style.css

```diff
@@ -1549,11 +1549,6 @@ form .post {
 
 @media (max-width: 920px),
 (display-mode: standalone) {
-    div.columns {
-        height: calc(100vh - 50px);
-        align-items: stretch;
```

Comment:
`align-items: stretch;` can be deleted safely because the same one is defined here: https://github.com/jointakahe/takahe/pull/370/files#diff-a5ea33c888430601a659bcbef2da1944097953737f2606282efc13ca6e5fbabaL338